how-to-docs/kubernetes-token.md

```diff
@@ -0,0 +1,15 @@
+# How to get a kubernetes token for the UK cluster
+
```

Comment:
capitalise Kubernetes, it's a proper noun

Reply:
Thank you